features/unique_email.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Unique Emails
+In order to identify my users via e-mail addresses (for example during login processes)
+As an admin
+I want to have only one user per e-mail address
+
+  Scenario: Sign up with existing email address
+    Given a user a with email test@test.com
+    And a new user b with email test@test.com
+    When b tries to sign up
+    Then b should not be able to sign-up
```

Comment:
Wording: maybe 'the sign up should fail'? It's also sign up, not sign-up.

Reply:
@Markus-Brand says: This needs a reference to a user, so it needs to include a reference to the name b

And yes, it is, will fix.